api/sql/schema/750-rule.decision.fetch.sql

```diff
@@ -3,7 +3,7 @@ ALTER PROCEDURE [rule].[decision.fetch]
     @operationDate DATETIME, -- the date when operation is triggered
     @sourceAccountId NVARCHAR(255), -- source account id
     @destinationAccountId NVARCHAR(255), -- destination account id
-    @amount MONEY, -- operation amount
+    @amount VARCHAR(21), -- operation amount
```

Comment:
After that you should define variable that is of type money or decimal, otherwise these checks below won't be correct:

@amount < l.minAmount OR
            @amount > l.maxAmount OR
            @amount + ISNULL(c.amountDaily, 0) > l.maxAmountDaily OR
            @amount + ISNULL(c.amountWeekly, 0) > l.maxAmountWeekly OR
            @amount + ISNULL(c.amountMonthly, 0) > l.maxAmountMonthly OR

Reply:
замених ги с DECIMAL(12, 2) и получихме желания ефект